.github/workflows/docker.yml

```diff
@@ -13,106 +15,136 @@ env:
     LCLI_IMAGE_NAME: ${{ github.repository_owner }}/lcli
 
 jobs:
-    extract-branch-name:
+    # Extract the VERSION which is either `latest` or `vX.Y.Z`, and the VERSION_SUFFIX
+    # which is either empty or `-unstable`.
+    #
+    # It would be nice if the arch didn't get spliced into the version between `latest` and
```

Comment:
This would be nice. We could update the naming without breaking backwards compatibility pretty easily by just continuing to push tags using the old naming convention.

That would only be two extra tags at the moment ->

`latest-arm64-unstable`
`latest-amd64-unstable`

After this PR we'd have to push some more, so it might be a now or never type of deal. I'm OK either way, seems like this PR is working well as-is.

Reply:
Having spent a while fiddling with the tags I'll vote for leaving it as-is for now :sweat_smile:

I think it's not too unsightly, and we can come back to clean it up at some point if we really want